app/templates/events/trackVolunteers.html

```diff
@@ -39,16 +39,19 @@ <h1 class="text-center">{{event.name}}</h1>
 <input class="form-control" id="eventID" name="event" value="{{event}}" type='hidden' />
 <div class="flex-container" style="padding: 5px;">
   <div class='form-group d-inline-flex'>
-    <a class="btn btn-success" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addVolunteerModal" style="margin:5px;">Add volunteer</a>
+    <a class="btn btn-success" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addVolunteerModal" style="margin:5px;">Add Volunteer</a>
     {% if event.isRecurring and not event.isFirstRecurringEvent%}
-      <a class="btn btn-success" id="pastVolunteers" role="button" data-bs-toggle="modal" style="margin:5px;">Add previous volunteers</a>
-    {% endif %}
+        {% if recurringVolunteers | count == 0  %}
+            {% set disabled = "disabled" %}
+        {% endif %}
+        <button type="button" class="btn btn-success" id="pastVolunteers" data-bs-toggle="modal" data-bs-target= "#addPastVolunteerModal" style="margin:5px;" {{disabled}}>Add Past Volunteers</button> 
```

Comment:
Avoid style tags if you can do a similar bootstrap utility. https://getbootstrap.com/docs/5.0/utilities/spacing/